config/main.py

```diff
@@ -18,10 +18,21 @@
 import aaa
 import mlnx
 
+import yaml
+import re
+from collections import OrderedDict
+import ast
+
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help', '-?'])
 
 SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
 SYSLOG_IDENTIFIER = "config"
+SNMP_COMMUNITY_FILE = '/etc/sonic/snmp.yml'
+SPEED_TYPE= ['100G', '50G']
+
+## TODOFIX: Need to change the path of this file 
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
+BREAKOUT_CFG_FILE = PLATFORM_ROOT_PATH + '/platform.json'
```

Comment:
Yes, will change the logic like below.
```
from sfputil.main import get_path_to_port_config_file
BREAKOUT_CFG_FILE = get_path_to_port_config_file()
```
once the sfputil file gets compatible with platform.json. 
> creating different PR to read the **_platform.json_** file instead of **_port_config.ini_** file

Reply:
addressed